.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
### Your Main Program

![image](https://user-images.githubusercontent.com/93039359/235134516-994a5026-250f-490e-9a42-5e909248367d.png)

### Design Main Program

![image](https://user-images.githubusercontent.com/93039359/235134654-c24c6797-e8aa-423d-a4e6-8b0c72aed943.png)

- [ ] Great work on making the changes but as suggested by the previous reviewer kindly make your boxes as same as they are in the given design :+1: to make your project look attractive and professional :blush: Your div height is more than the design div :fearful: Kindly fix it :thinking: